website/docs/operations.md

```diff
@@ -153,6 +153,29 @@ At a high level, this requires:
 * The ability to write to all objects in the group `mutations.gatekeeper.sh` (mutators)
 * The ability to read `MutatorPodStatus` objects in Gatekeeper's namespace
 
+## Mutation Controller
+
+__--operation key:__ `mutation-controller`
+
+This operation runs the process responsible for ingesting and registering
```

Comment:
For UX of folks using gatekeeper, would it be worth `WARN` logging when operators turn on two redundant `operation` knobs here? Like when someone specific both `mutation-webhook` and `mutation-controller` ?

Reply:
No, there is no cost to using redundant flags.